components/Tabs/tests/unit/MDCTabBarColorSchemeTests.m

```diff
@@ -0,0 +1,58 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialTabs.h"
+#import "MaterialThemes.h"
+#import "MDCTabBarColorThemer.h"
+
+@interface MDCTabBarTestColorScheme : NSObject <MDCColorScheme>
+
+@end
+
+@implementation MDCTabBarTestColorScheme
+
+- (UIColor *)primaryColor {
+  return [UIColor redColor];
+}
+
+- (UIColor *)primaryDarkColor {
+  return [UIColor darkGrayColor];
+}
+
+- (UIColor *)primaryLightColor {
+  return [UIColor yellowColor];
+}
+
+@end
+
+@interface MDCTabBarColorSchemeTests : XCTestCase
```

Comment:
Naming: This looks like an MDCTabBarColorThemer test, not a scheme test. Consider renaming to `MDCTabBarColorThemerTests`